src/main/java/de/lh/tool/domain/model/UserRole.java

```diff
@@ -0,0 +1,126 @@
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+@Table(name = "user_role")
+public class UserRole implements GrantedAuthority {
+
+	private static final long serialVersionUID = -370455906327675533L;
+
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	private Long id;
+
+	@OneToOne
+	@JoinColumn(insertable = true, name = "user_id", unique = true, updatable = true)
+	private User user;
+
+	@Column(name = "role", length = 100, nullable = false)
+	private String role;
+
+//  ██████╗__██████╗_██╗_____███████╗███████╗
+//  ██╔══██╗██╔═══██╗██║_____██╔════╝██╔════╝
+//  ██████╔╝██║___██║██║_____█████╗__███████╗
+//  ██╔══██╗██║___██║██║_____██╔══╝__╚════██║
+//  ██║__██║╚██████╔╝███████╗███████╗███████║
+//  ╚═╝__╚═╝_╚═════╝_╚══════╝╚══════╝╚══════╝
+
+	public static final String ROLE_ADMIN = "ROLE_ADMIN";
+	public static final String ROLE_CONSTRUCTION_SERVANT = "ROLE_CONSTRUCTION_SERVANT";
+	public static final String ROLE_LOCAL_COORDINATOR = "ROLE_LOCAL_COORDINATOR";
+	public static final String ROLE_SERVICE_COMMITTEE = "ROLE_SERVICE_COMMITTEE";
+	public static final String ROLE_PUBLISHER = "ROLE_PUBLISHER";
+	public static final String ROLE_STORE_KEEPER = "ROLE_STORE_KEEPER";
+	public static final String ROLE_INVENTORY_MANAGER = "ROLE_INVENTORY_MANAGER";
+
+//  ██████╗_██╗_██████╗_██╗__██╗████████╗███████╗
+//  ██╔══██╗██║██╔════╝_██║__██║╚══██╔══╝██╔════╝
+//  ██████╔╝██║██║__███╗███████║___██║___███████╗
+//  ██╔══██╗██║██║___██║██╔══██║___██║___╚════██║
+//  ██║__██║██║╚██████╔╝██║__██║___██║___███████║
+//  ╚═╝__╚═╝╚═╝_╚═════╝_╚═╝__╚═╝___╚═╝___╚══════╝
+//	Unfortunately granted authorities always have to start with "ROLE_". Several attempts to change that failed. 	
+	public static final String RIGHT_USERS_GET_ALL = "ROLE_RIGHT_USERS_GET_ALL";
+	public static final String RIGHT_USERS_CREATE = "ROLE_RIGHT_USERS_CREATE";
+
+	private static class RoleRightManager {
+		private static RoleRightManager instance;
+		private final Map<String, Collection<String>> roleRights;
+
+		private RoleRightManager() {
+			roleRights = new HashMap<>();
+
+//  _██████╗_██████╗__█████╗_███╗___██╗████████╗██╗███╗___██╗_██████╗_
+//  ██╔════╝_██╔══██╗██╔══██╗████╗__██║╚══██╔══╝██║████╗__██║██╔════╝_
+//  ██║__███╗██████╔╝███████║██╔██╗_██║___██║___██║██╔██╗_██║██║__███╗
+//  ██║___██║██╔══██╗██╔══██║██║╚██╗██║___██║___██║██║╚██╗██║██║___██║
+//  ╚██████╔╝██║__██║██║__██║██║_╚████║___██║___██║██║_╚████║╚██████╔╝
+//  _╚═════╝_╚═╝__╚═╝╚═╝__╚═╝╚═╝__╚═══╝___╚═╝___╚═╝╚═╝__╚═══╝_╚═════╝_
+			roleRights.put(ROLE_ADMIN, Arrays.asList(RIGHT_USERS_GET_ALL, RIGHT_USERS_CREATE));
+			roleRights.put(ROLE_CONSTRUCTION_SERVANT, Arrays.asList());
+			roleRights.put(ROLE_LOCAL_COORDINATOR, Arrays.asList());
+			roleRights.put(ROLE_SERVICE_COMMITTEE, Arrays.asList());
+			roleRights.put(ROLE_PUBLISHER, Arrays.asList());
+			roleRights.put(ROLE_STORE_KEEPER, Arrays.asList());
+			roleRights.put(ROLE_INVENTORY_MANAGER, Arrays.asList());
+		}
```

Comment:
Kleinigkeit: Warum hier nicht das schöne neue List.of() bzw auch Collections.emptyList() verwenden...